report/src/main/webapp/app/layout/main/suite/mainView.suite.view.html

```diff
@@ -26,17 +26,11 @@
         <div class="inner">
           <div ng-class="test.getStatus()">
             <div class="test-count test-count-dashboard">
-              <div ng-if="test.getStatus() == 'failed'" class="failed">
-                <i class="fas fa-times"></i>
-              </div>
-              <div ng-if="test.getStatus() == 'passed'">
-                <i class="fas fa-check"></i>
-              </div>
-              <div ng-if="test.getStatus() == 'warning'">
-                <i class="fas fa-exclamation-triangle"></i>
+              <div ng-if="test.getStatus() != 'rebased'" class="failed">
+                <i class="fas {{theme.statusClasses[test.getStatus()]}}"></i>
               </div>
               <div ng-if="test.getStatus() == 'rebased'">
-                <i class="fas fa-cloud-upload-alt"></i>
+                <i class="fas {{theme.statusClasses[test.getStatus()]}}"></i>
```

Comment:
Those elements are required, so that when a case gets accepted/rejected the icon is redrawn - simply adding an element with {{theme.statusClasses[test.getStatus()]}} won't work, since the italic tags are replaced with svg tags by fontawesome.